### PR TITLE
Refactore SSL assets

### PR DESCRIPTION
**Pull request by lennerd**
Related to #93

### CODE REVIEW

**Comment by lennerd** [comment]
@Xylakant I deployed this PR on staging, but it still does not use the correct protocol.

**Comment by Xylakant** [comment]
the config change was deployed on live only, let me fix stage.

**Comment by Xylakant** [comment]
try again please.

**Comment by lennerd** [comment]
Looks good, thank you! 👍

@1000miles Please review so we can merge this into master finally.

File: config/environments/staging.rb [hunk 1]
```diff
@@ -54,7 +54,7 @@
   config.serve_static_assets = false
 
   # Enable serving of images, stylesheets, and javascripts from an asset server
-  config.action_controller.asset_host = "http://staging.wheelmap.org"
+  config.action_controller.asset_host = "staging.wheelmap.org"
```
Comment (1000miles) [inline config/environments/staging.rb]: @lennerd I guess this looks good to me to remove the protocol. 

I have a question:
1. Do we use an ssl-encryption package that is valid for all pages on wheelmap.org or are we somehow limited to some pages that can not use `https`? 

If yes, only ssl - then we could use  `config.force_ssl = true`, or?

Comment (lennerd) [inline config/environments/staging.rb]: I'm not quite sure. also about forcing to use SSL. @Xylakant @holgerd do you have information here?

Comment (1000miles) [inline config/environments/staging.rb]: @lennerd thanks. okay, it might be another task. we can postpone this question to a later time. :)

**Comment by 1000miles** [comment]
@lennerd Thank you!

**Pull request merged**